qiskit/circuit/library/quantum_volume.py

```diff
@@ -83,39 +83,48 @@ def __init__(
         depth = depth or num_qubits  # how many layers of SU(4)
         width = num_qubits // 2  # how many SU(4)s fit in each layer
         rng = seed if isinstance(seed, np.random.Generator) else np.random.default_rng(seed)
+        seed_name = seed
         if seed is None:
```

Comment:
IMO it would be clearer to check if `seed_name` is `None` instead, since the purpose of this branch is to make sure it has a value.